apps/deploy-web/src/components/alerts/ContactPointsListContainer/ContactPointsListContainer.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import type { ReactNode } from "react";
+import { type FC, useCallback, useState } from "react";
+import type { components } from "@akashnetwork/react-query-sdk/notifications";
+
+import { useServices } from "@src/context/ServicesProvider";
+import { useNotificator } from "@src/hooks/useNotificator";
+import { useWhen } from "@src/hooks/useWhen";
+
+export type ContactPoint = components["schemas"]["ContactPointOutput"]["data"];
+export type ContactPointsInput = components["schemas"]["ContactPointListOutput"]["data"];
+
+export type ContactPointsListViewProps = {
+  data: ContactPointsInput;
+  totalCount: number;
+  page: number;
+  limit: number;
+  isLoading: boolean;
+  edit: (id: ContactPoint["id"]) => void;
```

Comment:
this is basically a callback, better be called
```suggestion
  onEdit: (id: ContactPoint["id"]) => void;
```

so on the client code it's
```tsx
<ContactPointsListContainer onEdit={doSmth} />
```

Reply:
Done